### DISCUSSION TITLE
# hashtag Cardinal (social media)

### DISCUSSION THREAD

**Discussion by subderground** [DISTRHO/Cardinal#435] [Ideas]
Just a 2cents question ;-)
For social medias, is there an "official" or  "favorite" #hashtag about Cardinal ? 
ie: #cardinalrack #cardinalmodular #cardinalmodularrack #cardinalvcv #cardinaldistrho vcvcardinal ect...

Nb: A cardinal is a high member of the catholic church, or a red bird. So "#cardinal" will not 100% match ;-)

**Comment by dromer** [comment]
#cardinalplugin perhaps?

**Reply by subderground** [reply]
The things I've posted before with  #cardinal are lost with a lot of cardinal birds xD

So I think **#cardinalplugin** & **cardinalrack** * are the best :-)

*Used by Steve Parker.

**Reply by falkTX** [reply]
personally I prefer #cardinalsynth

**Comment by GBaige** [comment]
#cardinalvst

**Reply by dromer** [reply]
but it's not just a vst, so it doesn't represent the whole project

**Reply by GBaige** [reply]
I reckon it doesn't need to; most users will look it up as "vst" anyway since it's the most common plugin format

**Reply by falkTX** [reply]
I am looking forward to the day where "vst" is seen in bad light, some old compat thing, and we have another generic term to refer to audio plugins.
